gaming/schema/players.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE games (
```

Comment:
Are players.sql and trading.sql the main schema files. It's unclear to me what 00001...00004.sql are. Can we avoid the latter (assuming they are some how generated) ?